package-lock.json

```diff
@@ -4089,6 +4090,910 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8="
     },
+    "fsevents": {
```

Comment:
It looks like the fsevents package, et. al. are being added again. I was able to get rid of it by `rm -rf node-modules`, bringing the container down, bringing the container up, and letting the container do the npm install, and that seemed to resolve it for me.

Reply:
Also make sure you have NPM 5.6.0. It was supposedly fixed in that version, but there is still [an issue open saying that it still doesn't work sometimes](https://github.com/npm/npm/issues/17722)...